ebml.xml

```diff
@@ -0,0 +1,61 @@
+<EBMLSchema xmlns="urn:ietf:rfc:8794" docType="ebml" version="1">
+	<element name="EBML" path="\EBML" id="0x1A45DFA3" type="master" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">Set the EBML characteristics of the data to follow. Each EBML document has to start with this.</documentation>
+	</element>
+	<element name="EBMLVersion" path="\EBML\EBMLVersion" id="0x4286" type="uinteger" range="1-8" default="1" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The version of EBML parser used to create the file.</documentation>
+	</element>
+	<element name="EBMLReadVersion" path="\EBML\EBMLReadVersion" id="0x42F7" type="uinteger" range="1" default="1" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The minimum EBML version a parser has to support to read this file.</documentation>
+	</element>
+	<element name="EBMLMaxIDLength" path="\EBML\EBMLMaxIDLength" id="0x42F2" type="uinteger" range="4" default="4" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The maximum length of the IDs you'll find in this file (4 or less in Matroska).</documentation>
+	</element>
+	<element name="EBMLMaxSizeLength" path="\EBML\EBMLMaxSizeLength" id="0x42F3" type="uinteger" range="1-8" default="8" minOccurs="1" maxOccurs="1">
+		<documentation lang="en" purpose="definition">The maximum length of the sizes you'll find in this file (8 or less in Matroska). This does not override the element size indicated at the beginning of an element. Elements that have an indicated size which is larger than what is allowed by EBMLMaxSizeLength shall be considered invalid.</documentation>
+	</element>
+	<element name="DocType" path="\EBML\DocType" id="0x4282" type="string" default="ebml" minOccurs="1" maxOccurs="1">
```

Comment:
missing `length=">0"` but in a proper XML way, so `length="&gt;0"`

Reply:
ok